.github/workflows/check-devcontainer.yml

```diff
@@ -0,0 +1,43 @@
+# This file is maintained by velocitas CLI, do not modify manually. Change settings in .velocitas.json
+# This file is maintained by velocitas CLI, do not modify manually. Change settings in .velocitas.json
```

Comment:
Duplicated header, coming from the additional line in the devenv-github-workflows package. See finding https://github.com/eclipse-velocitas/devenv-github-workflows/pull/40#discussion_r1311249523